style.css

```diff
@@ -0,0 +1,271 @@
+    width: 8rem;
+}
+.brand-tagline{
+    font-size: 1.5rem;
+    color: var(--primary-color);
+    margin:1rem 0;
+    letter-spacing: 0.1rem;
+}
+.btn{
+    padding-top: 0.8rem !important;
+    padding-bottom: 0.8rem !important;
+}
+.main{
+    width: 100vw;
+}
+.side-bar-navigation{
+    position: fixed;
+    min-width: max-content;
+    height: 100vh;
+    top: 0;
+    left:0;
+}
+.side-bar{
+    display: flex;
+    flex-direction: column;
+    background: #f7f7f7;
+    height: max-content;
+    height: 100%;
+    padding: var(--padding-md);
+    padding-right: 0;
+    z-index: 2;
+}
+#brand{
+    margin-bottom: 1rem;
+}
+.heading-md:hover{
+    color: var(--txt-light);
+}
+.component-list__container{
+    display: flex;
+    flex-direction: column;
+}
+
+.component-item{
+    color: var(--txt-dark);
+    padding:var(--padding-sm);
+    border-radius: 0.5rem;
+    width: max-content;
+    cursor: pointer;
+    list-style: none;
+    transition: 0.2s;
+}
+
+.component-item:hover{
+    transform: scale(1.1);
+    background: var(--secondary-color);
+    color: var(--txt-primary);
+}
+.docs__container{
+    width: 100vw;
+    padding-left: 20%;
+    min-height: 100vh;
+}
+#usage,#avatar,#alert,#button,#card,#image,#navbar,#toast,#typography{
+    padding-top: 1rem;
+}
+#usage{
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+.docs-para{
+    font-size: 1.1rem;
+    margin: 0.5rem 0;
+}
+.code{
+    background: var(--txt-light);
+    color: var(--txt-dark);
+    padding: 0.2rem;
+    border-radius: 0.2rem;
+    display: inline;
+}
+.code-snippet__container{
+    width: 95%;
+    height: max-content;
+    overflow-x: auto;
+    margin: 0.5rem 0;
+}
+.code-snippet{
+    width: 80%; 
+    padding: 0;
+    border:0; 
+    display: block;
+    z-index: 1;
+}
+.component-preview__container{
+    display: grid;
+    width:max-content;
+    padding: var(--padding-sm);
+    border-radius:0.5rem;
+    box-shadow: 0 4px 5px -2px var(--secondary-color);
+    grid-template-columns: repeat(3 , 1fr);
+    grid-gap: 0.5rem;
+}
+.component-preview__container-flex{
+    display: flex;
+    width:80%;
+    justify-content: space-evenly;
+    padding: var(--padding-sm);
+    border-radius:0.5rem;
+    box-shadow: 0 4px 5px -2px var(--secondary-color);
+}
+#typo-flex{
+    flex-direction: column;
+}
+@media only screen and (max-width:1050px){
+    .docs__container{
+        padding-left: 23%;
+    }
+}
+
+@media only screen and (max-width:930px){
+    .docs__container{
+        padding-left: 30%;
+    }
+    .component-preview__container{
+        grid-template-columns: repeat(2,1fr);
+    }
+    .component-preview__container-flex{
+        flex-direction: column;
+    }
+}
+@media only screen and (max-width: 700px) {
+    .header{
+        display: block;
+        position: fixed;
+        top: 0;
+        z-index: 2;
+    }
+    #brand{
+        display: none;
+    }
+    .side-bar-navigation{
```

Comment:
Here as well, you could take a look at the class name pattern to maintain consistency.

Reply:
Fixed, Do check it out